### PR TITLE
Making changes as requested in issue #124

### PR DESCRIPTION
**Pull request by astrilet**

### CODE REVIEW

**Comment by astrilet** [comment]
Fixed "Terms of Use" link

File: app/views/layouts/_dmponline_footer.html.erb [hunk 1]
```diff
@@ -10,14 +10,14 @@
       <% end %>
 
    			</p>
-       	
-    </div>	
+
+    </div>
    	<div class="right_side_footer">
-   		<br/> <br/> <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">Web Site Privacy Policy</a> | <%= link_to t('helpers.terms'), terms_path %>
+        <br/> <br/> <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">Web Site Privacy Policy</a> | <a href='terms'>Terms and conditions</a>
```
Comment (weiweishi) [inline app/views/layouts/_dmponline_footer.html.erb]: Sorry @astrilet The url to terms need to be replaced by this link https://portagenetwork.ca/portage-dmp-assistant-terms-use/

File: config/routes.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
+TERMS_OF_USE_URL='https://portagenetwork.ca/portage-dmp-assistant-terms-use/'
 DMPonline4::Application.routes.draw do
  get '/help', to: redirect("/#{I18n.default_locale}/help")
  get '/terms', to: redirect("/#{I18n.default_locale}/terms")
```
Comment (weiweishi) [inline config/routes.rb]: this has not been updated?

Comment (astrilet) [inline config/routes.rb]: just updated it

**Comment by weiweishi** [comment]
Double checked. The French version of the terms and conditions link should goes to:
https://portagenetwork.ca/fr/conditions-dutilisation-de-lassistant-pgd-de-portage/

I will create a different issue, but it seems that the footer doesn't change based on locale.

**Pull request merged**